knowledge-base/actions/bazelbuild/setup-bazelisk/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Setup Bazelisk'
+github-token:
```

Comment:
There is no need for adding permissions for this KB because if there is only `content: read` permission, then there is no need of adding permission as it gets added at workflow level by default.